### ISSUE TITLE
Moneris support

### ISSUE THREAD

**Issue by dwenaus** [thephpleague/omnipay#425]
I see a few libraries that seem to support Moneris. 
https://github.com/kbahchevanov/omnipay-moneris
https://github.com/darksaboteur/omnipay-moneris
Should these be added in to the main account, are the fine to use?

**Comment by delatbabel** [comment]
Yes, they can be added in.  Encourage one of the owners to submit a PR here.
